qldpc/decoder.py

```diff
@@ -119,8 +121,8 @@ def _build_cvxpy_constraints(
 
     If `lower_bound_row is not None`, treat the constraint at this row index as a lower bound.
     """
-    matrix = matrix % modulus
-    syndrome = syndrome % modulus
+    matrix = np.array(matrix) % modulus
+    syndrome = np.array(syndrome) % modulus
```

Comment:
Without this change, [this](https://github.com/Infleqtion/qLDPC/blob/826ecd93a2f8340579a83f247dc88bf2392d4221/qldpc/codes/common_test.py#L315) test assertion fails.  I think this change is necessary due to the `numpy` downgrade (which I think is in turn required by the `sinter` dependency of `ldpc`).